Resources/doc/serialization-groups-and-relations.md

```diff
@@ -0,0 +1,160 @@
+# Serialization groups and relations
+
+## Using serialization groups
+
+Symfony 2.7 introduced [serialization (and deserialization) groups support](http://symfony.com/blog/new-in-symfony-2-7-serialization-groups)
+in the Serializer component. Specifying to the API system the groups to use is damn easy:
+
+```yaml
+services:
+    resource.product:
+        parent:    "api.resource"
+        arguments: [ "AppBundle\Entity\Product" ]
+        calls:
+            -      method:    "initNormalizationContext"
+                   arguments:
+                       -      { groups: [ "serialization_group1", "serialization_group2" ] }
```

Comment:
Yes, in some cases (especially with custom normalizers) you want to pass other data to the `$context` argument than groups (e.g. the XML serializer of Symfony allows to choose the name of the root XML tag).